File: problems/algorithms/date-diff-timestamp/js/response/diff-timestamp.test.js
import { diffMonth } from './diff-timestamp.js';

describe('Date Tool', () => {
  test('calculating the difference between months', () => {
    let start = 1483239600000; // (new Date(2017, 0, 1)).getTime()
    let end = 1496762425846; // Date.now()
    expect(diffMonth(start, end)).toBe(5);
  });
});

File: problems/algorithms/date-diff-timestamp/ruby/response/spec/diff-timestamp.test.rb
require_relative "../src/diff-timestamp"

describe DateDiff do
    describe ".compute" do
       context "With the 1483239600000 as start and 1496762425846 as end month" do
        it "return 5" do
            expect(DateDiff.compute(1483239600000, 1496762425846)).to eql(5)
        end
    end
end